admin/src/templates/export_user-devices.html

```diff
@@ -0,0 +1,9 @@
+<div class="col-sm-12">
+  <div>
+    <p translate>export.user-devices.description</p>
+    <a class="btn btn-default" ng-click="export()" ng-disabled="exporting">
+      <i class="fa fa-arrow-down"></i>
+      <span translate>Download</span>
```

Comment:
Same here about translation keys. Like in L3 to keep consistency: `export.button.download`